bundles/io/org.eclipse.smarthome.io.transport.mqtt/src/main/java/org/eclipse/smarthome/io/transport/mqtt/MqttService.java

```diff
@@ -34,7 +39,9 @@
  * @author Davy Vanherbergen
  * @author Markus Rathgeb - Synchronize access to broker connections
  */
-public class MqttService implements ManagedService {
+@Component(immediate = true, service = { MqttService.class }, configurationPid = {
+        "org.eclipse.smarthome.mqtt" }, name = "MQTTConnectionService", property = "service.pid=org.eclipse.smarthome.mqtt")
```

Comment:
WRT to #4113, should we set the name ever?

Reply:
@kaikreuzer WDYT?

Reply:
No, we shouldn't.